be1-go/channel/election/verification.go

```diff
@@ -122,6 +123,22 @@ func (c *Channel) verifyMessageCastVote(castVote messagedata.VoteCastVote) error
 		return xerrors.Errorf("cast vote created at is %d, should be minimum 0", castVote.CreatedAt)
 	}
 
+	for i, vote := range castVote.Votes {
+		qs, ok := c.questions[vote.Question]
+		if !ok {
+			return xerrors.Errorf("no Question with question ID %s exists", vote.Question)
+		}
+
+		sort.Ints(vote.Vote)
+		sortedIndex := arrayToString(vote.Vote, ",")
+
+		hash := messagedata.Hash("Vote", electionID, string(qs.id), sortedIndex)
+
+		if vote.ID != hash {
+			return xerrors.Errorf("vote ID of vote %d is %s should be %s", i, vote.ID, hash)
```

Comment:
Isn't it weird to have sometimes id in lowercase and sometimes ID in uppercase? 

I know that it isn't probably you who wrote that, but if it isn't too complicated I think it could be great to be consistent.
What do you think?